packages/datatrak-web/src/api/queries/useUser.ts

```diff
@@ -6,17 +6,21 @@
 import { useQuery } from 'react-query';
 import { DatatrakWebUserRequest } from '@tupaia/types';
 import { get } from '../api';
+import { useProjects } from './useProjects.ts';
 
 export const useUser = () => {
   const userResponse = useQuery(
     'getUser',
     (): Promise<DatatrakWebUserRequest.ResBody> => get('getUser'),
   );
+  const { data: projects } = useProjects();
+
   const { data: user } = userResponse;
+  const userProject = projects?.find(({ id }) => id === user?.projectId);
 
   return {
     ...userResponse,
-    data: { ...user, name: user?.userName },
+    data: { ...user, name: user?.userName, project: userProject },
```

Comment:
Probably when my request project access PR is merged in this will need to be removed